src/matlab.ts

```diff
@@ -54,7 +54,7 @@ export async function runCommand(hs: HelperScript, platform: string, architectur
     const rmcPath = getRunMATLABCommandScriptPath(platform, architecture);
     await fs.chmod(rmcPath, 0o777);
 
-    const rmcArg = script.cdAndCall(hs.dir, hs.command);
+    const rmcArg = `setenv("MW_ORIG_WORKING_FOLDER", cd('${hs.dir}'));${hs.command}`;
```

Comment:
Safest to use `'` instead of `"` to support MATLAB releases as far back as possible.

Also probably a good idea to escape any `'` chars in `hs.dir`.